tests/helper_test.py

```diff
@@ -0,0 +1,32 @@
+# encoding=utf-8
+
+"""Test suite for alot.helper module."""
+
+import unittest
+
+from alot.helper import shorten_author_string
+
+
+class TestHelperShortenAuthorString(unittest.TestCase):
+
+    authors = u'King Kong, Mucho Muchacho, Jaime Huerta, Flash Gordon'
+
+    def test_high_maxlength_keeps_string_intact(self):
+        self.assertEqual(shorten_author_string(self.authors, 60), self.authors)
```

Comment:
I'm terrible about doing this myself, but... It's better to assign the value of the function being tested than to pass it directly into the assert, since that makes it much easier to inspect with a debuger like pdb if the test is failing.